config/externalname.go

```diff
@@ -2281,6 +2281,8 @@ var TerraformPluginSDKExternalNameConfigs = map[string]config.ExternalName{
 	"aws_sagemaker_image": config.ParameterAsIdentifier("image_name"),
 	// SageMaker Code Images can be imported using the name
 	"aws_sagemaker_image_version": config.IdentifierFromProvider,
+	// Sagemaker MLFlow tracking server can be imported using the name
+	"aws_sagemaker_mlflow_tracking_server": config.NameAsIdentifier,
```

Comment:
To use `NameAsIdentifier` as an external name, the resource must be imported with the name, and the name argument must be present. See https://github.com/crossplane/upjet/blob/main/docs/adding-new-resource.md#case-1-name-as-identifier

When I look at the resource's [import document](https://registry.terraform.io/providers/hashicorp/aws/5.73.0/docs/resources/sagemaker_mlflow_tracking_server#import), it says that it can be imported with `workteam_name`. I assume there is a mistake here and that is because I don't see an argument called `workteam_name`. Instead, `tracking_server_name` can be used, and in this case, the most appropriate external name configuration is [ParameterAsIdentifier](https://github.com/crossplane/upjet/blob/main/docs/adding-new-resource.md#case-2-parameter-as-identifier).

Could you please regenerate the resource with the following configuration?

```suggestion
	"aws_sagemaker_mlflow_tracking_server": config.ParameterAsIdentifier("tracking_server_name"),
```

Reply:
I made the mistake of thinking `id` was acceptable for `NameAsIdentifier`

Thanks @turkenf. I've updated